### ISSUE TITLE
adjust order of components in principal edition display

### ISSUE THREAD

**Issue by paregorios** [DCLP/dclpxsltbox#191]
@jcowey raised the following issue during review of content associated with #145:

http://dclp.github.io/dclpxsltbox/output/dclp/100/99548.html
Principal Edition 514 P. Lond. Copt.
should be
Principal Edition P. Lond. Copt. 514

https://github.com/DCLP/idp.data/blob/dclp/DCLP/100/99548.xml#L88-L92

has changed because the biblio id

https://github.com/DCLP/idp.data/blob/dclp/DCLP/100/99548.xml#L89

has been added instead of

<title level="s" type="abbreviated">P. Lond. Copt.</title>
but the information is available from the file at

https://github.com/DCLP/idp.data/blob/dclp/DCLP/100/99548.xml#L92

Sorry. MEA CULPA. not transferring news about change to XML which has knock on effects for XSLT.

Would you please change the XSLT accordingly. That is read out
content of
https://github.com/DCLP/idp.data/blob/dclp/DCLP/100/99548.xml#L91-L93

<title level="s" type="abbreviated">.+</title> before reading out content of https://github.com/DCLP/idp.data/blob/dclp/DCLP/100/99548.xml#L90 biblScope unit="number" etc

**Comment by jcowey** [comment]
Super thanks. And yes of course this is a separate issue

**Comment by leoba** [comment]
I have fixed this by changing a line in navigator/pn-xslt/metadata-dclp.xsl (which is in my checkout but not listed in https://github.com/DCLP/navigator/tree/master/pn-xslt for some reason). The default action had been to dump the value of everything in the "Principal Edition" (so if the title was listed after the number, that's how it would be displayed). So I broke it up to display `t:title` followed by `t:biblStruct['volume']` followed by `t:biblStruct['numbers']` - volume is not always present. I am committing these changes now, check line 203 in metadata-dclp.xsl and dclpxsltbox/output/dclp/100/99548.html and dclpxsltbox/output/dclp/1/555.html for examples.

**Comment by paregorios** [comment]
I'll take a look at this.

**Comment by paregorios** [comment]
hi @leoba I can't seem to find the change you're referencing. Can you provide a link to the file in github showing your change? metadata-dclp.xsl should not be in the "master" branch, but rather you should see it in "dclp-master".

**Comment by leoba** [comment]
Like a n00b, I neglected to push my commit. It's in there now:

https://github.com/DCLP/navigator/blob/xslt-master/pn-xslt/metadata-dclp.xsl#L203

**Comment by paregorios** [comment]
thanks @leoba

**Comment by jcowey** [comment]
I notice that @leoba pushed a commit. Did you @paregorios take that commit as a basis for new HTML files? I could not check on the xslt change, because the link
https://github.com/DCLP/navigator/blob/xslt-master/pn-xslt/metadata-dclp.xsl#L203
does not work (branch changes presumably caused this not to work)

**Comment by paregorios** [comment]
@jcowey yes, that was fixed and incorporated, but some of our other changes seem to have had an unpleasant side effect, see #212, which I hope to address shortly.

**Comment by paregorios** [comment]
This ticket can be closed in my opinion, but #212 should be left active.
